clients/banking/src/components/TransferWizardBeneficiary.tsx

```diff
@@ -54,15 +54,17 @@ export const TransferWizardBeneficiary = ({
   onSave,
 }: Props) => {
   const [ibanState, setIbanState] = useState<
-    | {
-        type: "ibanValidation";
-        data: AsyncData<GetIbanValidationQuery["ibanValidation"]>;
-      }
-    | {
-        type: "beneficiaryVerification";
-        data: AsyncData<GetBeneficiaryVerificationQuery["beneficiaryVerification"]>;
-      }
-  >();
+    AsyncData<
+      | {
+          type: "ibanValidation";
+          data: GetIbanValidationQuery["ibanValidation"];
+        }
+      | {
+          type: "beneficiaryVerification";
+          data: GetBeneficiaryVerificationQuery["beneficiaryVerification"];
+        }
```

Comment:
technically we should be covered by `__typename` so we might not need the `type`